chess-AI/experimental-work/NN Layout.py

```diff
@@ -1,83 +1,77 @@
         # Use 2 1x1 filters to convolve input channels to 2 output channels, one representing piece
-        # to move, and the other representing move to take out from possible moves.
-        # Use them to pick move from possible moves, represented by 73 channels of 8x8, or 4672 possible moves <- (Revising)
-        self.policyHead = nn.Sequential(nn.Conv2d(in_channels=256, out_channels=2, kernel_size=1),
-                                       nn.BatchNorm2d(2),
-                                       nn.ReLU(inplace=True),
-                                       nn.Flatten(),
-                                       nn.Linear(in_features=2*8*8, out_features=73*8*8))
-        
+        # to move, and the other representing move to take out of possible moves.
+        # Use them to pick move from possible moves, represented by 73 channels of 8x8, or 4672 possible moves
+        self.policy_head = nn.Sequential(nn.Conv2d(in_channels = 256, out_channels = 2, kernel_size = 1, bias = False),
+                                         nn.BatchNorm2d(2),
+                                         nn.ReLU(inplace = True),
+                                         nn.Flatten(),
+                                         nn.Linear(in_features = 2 * 8 * 8, out_features = 73 * 8 * 8))
+         
         # Convolve 256 8x8 channels into 8x8 channel, then use fully connected layer to take 64 input features from 8x8
         # channel and transform to 256 output features, then transform to one scalar value.
-        self.valueHead = nn.Sequential(nn.Conv2d(in_channels=256, out_channels=1, kernel_size=1),
-                                       nn.BatchNorm2d(1),
-                                       nn.ReLU(inplace=True),
-                                       nn.Flatten(),
-                                       nn.Linear(in_features=8*8, out_features=256),
-                                       nn.ReLU(inplace=True),
-                                       nn.Linear(in_features=256, out_features=1),
-                                       nn.Tanh())
+        self.value_head = nn.Sequential(nn.Conv2d(in_channels = 256, out_channels = 1, kernel_size = 1, bias = False),
+                                        nn.BatchNorm2d(1),
+                                        nn.ReLU(inplace = True),
+                                        nn.Flatten(),
+                                        nn.Linear(in_features = 8 * 8, out_features = 256),
+                                        nn.ReLU(inplace = True),
+                                        nn.Linear(in_features = 256, out_features = 1),
+                                        nn.Tanh())
 
         
-
     def forward(self, x):
-        x = self.convLayer(x)
+        x = self.conv_layer(x)
         
         # Go through all the residual blocks and add the input of the block to the output before regularization.
-        for numRes in range(3):
+        for num_res in range(self.num_res_blocks):
```

Comment:
Since you don't use `num_res` in this loop, you can do `for _ in range(self.num_res_blocks)` to make it clear that the index variable is not used.